tests/api2/test_261_iscsi_cmd.py

```diff
@@ -285,6 +285,21 @@ def zvol_resize(zvol, volsize, expected_status_code=200):
     assert result.status_code == expected_status_code, result.text
 
 
+def get_iscsi_sessions(payload=None, check_length=None):
+    results = GET('/iscsi/global/sessions', payload=payload)
```

Comment:
Why not use websocket calls here in `get_iscsi_sessions` and `get_client_count`?